test/jdk/java/foreign/TestHandshake.java

```diff
@@ -120,6 +124,13 @@ private void backoff() {
         }
     }
 
+    static void start(String name) {
+        if (started.compareAndSet(false, true)) {
+            long delay = System.currentTimeMillis() - start.get();
+            System.out.println("Started first thread: " + name + " ; delay (ms): " + delay);
```

Comment:
The additional diagnostic statements look good. Delay here is time "elapsed" since start right? (as opposed to the prior use of "delay" to mean "delay start of handshaker", right?

Reply:
Good point - I'm using the wrong term - I'll rectify this.